code/game/objects/items/devices/scanners.dm

```diff
@@ -298,6 +300,7 @@ REAGENT SCANNER
 /obj/item/device/mass_spectrometer/adv
 	name = "advanced mass spectrometer"
 	icon_state = "adv_spectrometer"
+	item_state = "adv_spectrometer"

```

Comment:
Does the /adv version not inherit the icon_state from its parent mass_spectrometer, which is set to "analyser" and need to be overwritten or does icon_state overwrite both inheritances?